feature: Add Code of Conduct and link in README

shikha-code404 opened the pull request:
Closes #55. Added the Contributor Covenant v2.1 as the CODE_OF_CONDUCT and linked it from the README, fulfilling the community guideline requirement.

thinkfast37 commented:
Content wise it is on point.  However, for the CoC to be actionable I believe the critical placeholders should be popuiated before the PR is merged.  Or this action needs to be performed in another ticket and that ticket should be linked in this PR.

shikha-code404 commented:
- @thinkfast37 and @vinaysingh282006,

Thank you for the review. I have addressed the feedback from @thinkfast37 regarding the critical placeholders in the `CODE_OF_CONDUCT.md` file.

**I have now:**
* Populated the main project email: `conduct@my-awesome-project.org`
* Added a secondary contact: `codeofconduct@githubusername.com`

The changes are confirmed to be working and deployed on the Netlify Preview.

Key changes:

- Made a CODE_OF_CONDUCT.md file .
- Gave link in the main Readme.md file of same file for easy access .
- Made changes according to the issue and updated my work .

Please let me know if this is now acceptable for final review and merging. 
close : #55
Thank you!
